From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: World of Warcraft <warcraft@users.noreply.github.com>
Date: Sun, 11 Oct 2026 16:19:22 +0000
Subject: [PATCH 2/6] G3D: include <sys/time.h> on every Unix

System.h uses struct timeval, which it only brought in on arm64 and FreeBSD. On x86_64 with musl
nothing else does.
---
 deps/g3dlite/include/G3D/System.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/g3dlite/include/G3D/System.h b/deps/g3dlite/include/G3D/System.h
index 10995247b1..5c9d33979c 100644
--- a/deps/g3dlite/include/G3D/System.h
+++ b/deps/g3dlite/include/G3D/System.h
@@ -20,7 +20,7 @@
 #include "G3D/BinaryFormat.h"
 #include "G3D/FileNotFound.h"
 #include <string>
-#if defined(__aarch64__) || defined(__FreeBSD__)
+#if !defined(_WIN32)
 #include <sys/time.h>
 #endif
 
